models/intel/product-detection-0001/model.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Product detector based on SSD-lite architecture with MobileNetV2 as a backbone
+  for Self-checkout Point of Sale related scenes. The detector is able to detect
+  12 classes of objects (sprite, kool-aid, extra, ocelo, finish, mtn_dew,
+  best_foods, gatorade, heinz, ruffles, pringles, del_monte)
+  and first labels 0 and 1 are related to background_label and undefined correspondingly.
+task_type: detection
+#files: (TBD)
+framework: dldt
+license: https://raw.githubusercontent.com/opencv/open_model_zoo/master/LICENSE
```

Comment:
Please remove the config for now. It will be generated by our internal infrastructure when we upload the model files.

Reply:
done